sympy/solvers/tests/test_solveset.py

```diff
@@ -1008,6 +1008,9 @@ def test_solveset():
     assert solveset(exp(x) - 1, x) == imageset(Lambda(n, 2*I*pi*n), S.Integers)
     assert solveset(Eq(exp(x), 1), x) == imageset(Lambda(n, 2*I*pi*n),
                                                   S.Integers)
+    # ISSUE 17940
+    _n = Dummy('n')
+    assert solveset(exp(exp(x))-5,x) == ImageSet(Lambda(_n, 2*_n*I*pi + log(log(5))), S.Integers)
```

Comment:
```suggestion
    assert solveset(exp(exp(x)) - 5, x) == ImageSet(Lambda(_n, 2*_n*I*pi + log(log(5))), S.Integers)
```

Reply:
Thanks a lot for suggestion and response.
:)

I think this should be discussed more, here the return type is different from all other similar problems.